src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/DataContent.cs

```diff
@@ -205,6 +206,22 @@ public ReadOnlyMemory<byte> Data
         }
     }
 
+    /// <summary>Gets the data represented by this instance as a Base64 character sequence.</summary>
+    /// <returns>The base64 representation of the data.</returns>
+    [JsonIgnore]
+    public ReadOnlyMemory<char> Base64Data
+    {
+        get
+        {
+            const string Base64Separator = ";base64,";
```

Comment:
FWIW, DataUriParser only looks for the comma.
https://github.com/dotnet/extensions/blob/fd92ff7497d30b0f4fda8d47bb0e6d0b33a3360c/src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/DataUriParser.cs#L37